docs/user/faq.rst

```diff
@@ -889,3 +889,35 @@ the tutorial in the docs provides an excellent introduction to
 `testing Falcon apps with pytest <http://falcon.readthedocs.io/en/stable/user/tutorial.html#testing-your-application>`_.
 
 (See also: `Testing <http://falcon.readthedocs.io/en/stable/api/testing.html>`_)
+
+How to set cookies in simulate request for testing?
+---------------------------------------------------
+
+This can be done by setting ``headers={'Cookies': 'xxx=yyy'}`` in ``simulate_request``. Here is an example:
+
+.. code:: python
+
+    import falcon
+    import falcon.testing
+    import pytest
+ 
+ 
+    class TastyCookies:
+ 
+        def on_get(self, req, resp):
+            resp.media = {'cookies': req.cookies}
+ 
+ 
+    @pytest.fixture
+    def client():
+        app = falcon.API()
```

Comment:
We should use [falcon.App](https://github.com/falconry/falcon/blob/master/falcon/app.py#L47), since `falcon.API` is deprecated, and will be removed in the next major release(4.0)

Reply:
solved.